webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -348,8 +360,10 @@ const PerformanceAnalysisTable: React.FC<Props> = ({
           ...pipelines.map<Column<Row>>((pipeline) => ({
             ...METRIC_COLUMN,
             field: `${pipeline}${metricName}`,
-            description,
-            ...groupHeader(pipeline, metricName),
+            ...groupHeader(
+              pipeline,
+              reactNodeWithTooltip(description, metricName)
```

Comment:
Continuing on https://github.com/ServiceNow/azimuth/pull/285#discussion_r1023030193
I don't like the `reactNode` thing. This function takes a `string` in argument and returns a `Typography` that has a tooltip. It is much more specific than dealing with `ReactNode`s.
Anyway, here is a different solution: I think the fact that it is so hard to find a name hints at the fact we don't need it. I would suggest:
```suggestion
              <Tooltip title={description}>
                <Typography variant="inherit">{metricName}</Typography>
              </Tooltip>
```

Reply:
Right, but the same lines of code, "Typography with the tooltip," is used in 2 different places in the code at - lines no# 365, 402. So, I would suggest we should keep this function and can decide on a proper name here. 

As you mentioned "reactNode" is not more appropriate here, we can use the names that you have suggested here "const typographyWithTooltip = (tooltip: string, typography: string) => (" earlier.

Reply:
I know. I just didn't think repeating 2 times 3 lines of trivial code can be deemed "dangerous" duplication. I would not consider it worth avoiding by creating a 5-line function. But it doesn't hurt either. Let's go with what you have right now. 👍